src/field/pokemon.ts

```diff
@@ -2842,20 +2853,14 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
 
     globalScene.applyModifiers(TempStatStageBoosterModifier, this.isPlayer(), Stat.ACC, userAccStage);
 
-    userAccStage.value = ignoreAccStatStage.value ? 0 : Math.min(userAccStage.value, 6);
+    userAccStage.value = ignoreAccStatStage.value ? 0 : Math.min(userAccStage.value, MAX_STAT_STAGE);
```

Comment:
```suggestion
    userAccStage.value = ignoreAccStatStage.value ? 0 : userAccStage.value;
```
If I'm following the logic correctly, the upper-bounding here is redundant with the clamping in `setStatStage`.

Reply:
I would agree. Since we are running  `const accuracyMultiplier = new NumberHolder(calcAccuracyMultiplier(userAccStage.value, targetEvaStage.value));`, which should take care of the clamping, we should be good